fhir-operation-bulkdata/src/main/java/com/ibm/fhir/operation/bulkdata/model/JobExecutionResponse.java

```diff
@@ -340,6 +340,11 @@ public Builder fhirStorageType(StorageDetail storageDetails) {
             jobParameter.setStorageDetails(storageDetails);
             return this;
         }
+
+        public Builder cosBucketNameOo(String cosBucketNameOo) {
```

Comment:
better use OperationOutcome instead of Oo?

Reply:
Fixed in next commit

Reply:
Fixed in next commit